data/simpy.gram

```diff
@@ -114,90 +110,90 @@ dotted_name[expr_ty]:
+    | 'for' t=star_targets 'in' ex=expressions ':' b=block {
         _Py_For(t, ex, b, NULL, NULL, EXTRA) }
 
 with_stmt[stmt_ty]:
-    | a=ASYNC 'with' '(' b=','.with_item+ ')' ':' c=block {
-        _Py_AsyncWith(b, c, NULL, EXTRA) }
-    | a=ASYNC 'with' b=','.with_item+ ':' c=block {
-        _Py_AsyncWith(b, c, NULL, EXTRA) }
-    | a='with' '(' b=','.with_item+ ')' ':' c=block {
-        _Py_With(b, c, NULL, EXTRA) }
-    | a='with' b=','.with_item+ ':' c=block {
-        _Py_With(b, c, NULL, EXTRA) }
+    | ASYNC 'with' '(' a=','.with_item+ ')' ':' b=block {
+        _Py_AsyncWith(a, b, NULL, EXTRA) }
+    | ASYNC 'with' a=','.with_item+ ':' b=block {
+        _Py_AsyncWith(a, b, NULL, EXTRA) }
+    | 'with' '(' a=','.with_item+ ')' ':' b=block {
+        _Py_With(a, b, NULL, EXTRA) }
+    | 'with' a=','.with_item+ ':' b=block {
+        _Py_With(a, b, NULL, EXTRA) }
 with_item[withitem_ty]:
     | e=expression o=['as' t=target { t }] { _Py_withitem(e, store_name(p, o), p->arena) }
 
 try_stmt[stmt_ty]:
-    | a='try' ':' b=block ex=except_block+ el=else_block f=finally_block {
+    | 'try' ':' b=block ex=except_block+ el=else_block f=finally_block {
         _Py_Try(b, ex, el, f, EXTRA) }
-    | a='try' ':' b=block ex=except_block+ el=else_block {
+    | 'try' ':' b=block ex=except_block+ el=else_block {
         _Py_Try(b, ex, el, NULL, EXTRA) }
-    | a='try' ':' b=block ex=except_block+ f=finally_block {
+    | 'try' ':' b=block ex=except_block+ f=finally_block {
         _Py_Try(b, ex, NULL, f, EXTRA) }
-    | a='try' ':' b=block ex=except_block+ {
+    | 'try' ':' b=block ex=except_block+ {
         _Py_Try(b, ex, NULL, NULL, EXTRA) }
-    | a='try' ':' b=block f=finally_block {
+    | 'try' ':' b=block f=finally_block {
         _Py_Try(b, NULL, NULL, f, EXTRA) }
 except_block[excepthandler_ty]:
-    | a='except' e=expression 'as' t=target ':' b=block {
+    | 'except' e=expression 'as' t=target ':' b=block {
         _Py_ExceptHandler(e, t->v.Name.id, b, EXTRA) }
-    | a='except' e=expression ':' b=block {
+    | 'except' e=expression ':' b=block {
         _Py_ExceptHandler(e, NULL, b, EXTRA) }
-    | a='except' ':' b=block {
+    | 'except' ':' b=block {
         _Py_ExceptHandler(NULL, NULL, b, EXTRA) }
 finally_block[asdl_seq*]: 'finally' ':' a=block { a }
 
 return_stmt[stmt_ty]:
-    | a='return' b=expressions { _Py_Return(b, EXTRA) }
-    | a='return' { _Py_Return(NULL, EXTRA) }
+    | 'return' a=expressions { _Py_Return(a, EXTRA) }
+    | 'return' { _Py_Return(NULL, EXTRA) }
 
 raise_stmt[stmt_ty]:
-    | a='raise' b=expression 'from' c=expression { _Py_Raise(b, c, EXTRA) }
-    | a='raise' b=expression { _Py_Raise(b, NULL, EXTRA) }
-    | a='raise' { _Py_Raise(NULL, NULL, EXTRA) }
+    | 'raise' a=expression 'from' b=expression { _Py_Raise(a, b, EXTRA) }
+    | 'raise' a=expression { _Py_Raise(a, NULL, EXTRA) }
+    | 'raise' { _Py_Raise(NULL, NULL, EXTRA) }
```

Comment:
This ought to be combinable into two:
```
    | 'raise' a=expression 'from' c=expression {...}
    | 'raise' a=[expression] {...}
```
(Alas, I can't figure out how to make it a single one. I filed #129 for a possible solution, but i think that would require significant refactoring of how the generated code for nested groups looks.)